index.html

```diff
@@ -9,14 +9,14 @@
     <title>Garvin Chimone</title>
 </head>
 <body>
-    <nav>
-        <div class="title"><h2>My Logo</h2></div>
+    <header>
+        <div class="title"><a href="index.html"><h2>My Logo</h2></a></div>
         <div class="burger-menu">
             <span class="bar"></span>
             <span class="bar"></span>
             <span class="bar"></span>
         </div>
-    </nav>
+    </header>
     
```

Comment:
- [ ] Actually what the previous reviewer meant was to wrap the whole navbar with the `header` tag for improved semantics. so kindly consider fixing it. the structure would look like this
```
<header>   
    <div class="title"><a href="index.html"><h2>My Logo</h2></a></div>
    <nav>
        <div class="burger-menu">
            <span class="bar"></span>
            <span class="bar"></span>
            <span class="bar"></span>
        </div>
    </nav>
</header>
```